morph_tool/axon_point.py

```diff
@@ -0,0 +1,67 @@
+"""Module to detect the terminal point of the main axon/"""
+import logging
+import numpy as np
+
+from neurom import COLS
+from morphio import SectionType, IterType
+
+L = logging.getLogger(__name__)
+
+
+def axon_point_section(morph, direction=None, bbox=None):
+    """Estimate axon point as the terminal point of the main axon.
```

Comment:
Shouldn't `main axon` be called `main axon trunk` or something like that? For me `main axon` supposes that there are several axons.

Reply:
I agree!!

Reply:
yes! What is the 'standard name' for that? `axon trunk`? To me `main` and `trunk `are synonymes, no?

Reply:
what I know is people use `collaterals` for the other branches

Reply:
I don't know what's the best but `axon trunk` would be already more clear (for me at least).

Reply:
@mzbili, shall we change the name of main axon? Or is it a standard name in litterature?

Reply:
it seems that the term main axon is used quite often, shall we stick to it?

Reply:
I am not an expert on this but it seems that 'main axon' is a shortcut for 'main axonal trunk' (here for example: http://www.informatics.jax.org/vocab/gene_ontology/GO:0044304). The first one seems widely used but the second one is more clear. But if 'main axon' is clear for the actual users of this we can keep it.

Reply:
ok, so main axon as a term is fine then, the trunk part is to explain what it is, if I read this page correctly.